examples/llama.py

```diff
@@ -218,6 +218,27 @@ def convert(name) -> Tensor:
     return lazy_tensors[0].cat(*lazy_tensors[1:], dim=axis)
   return {name: convert(name) for name in {name: None for model in models for name in model}}
 
+def load(fn:str):
+  if fn.endswith('.safetensors'):
+    from safetensors import safe_open
+    loader = safe_open(fn, framework="pt", device="cpu")
```

Comment:
can you use `safe_load` from `tinygrad/state` ?

Reply:
done, thanks!